remo/events/static/events/js/events_list.js

```diff
@@ -336,8 +336,8 @@ var update_results = function(data, query, newquery, past_events) {
     if (newquery) {
         clear_map();
         EventsLib.events_table_body_elm.empty();
-        set_number_of_events(data.meta.total_count);
     }
+    set_number_of_events(data.meta.total_count);
```

Comment:
we do we need to do this every time we update the results and not just when it's a new query?

Reply:
Because in few cases wasn't updated. On my tests I was trying to understand why the numbers wasn't showed

Reply:
Do you remember which cases? Then I can try to reproduce it.

Reply:
The case was on refresh of the page during my tests on this pr.
I think that is not a problem refresh that value on every query also if it is the same, is only a tiny dom operation that print that number.

Reply:
This just happened for me as well. So we're fine here with this change.